build-web-bundles.fsx

```diff
@@ -57,4 +57,4 @@ Target "Default" DoNothing
     ==> "Default"
 
 // start build
-RunParameterTargetOrDefault "target" "Default"
+RunTargetOrDefault "Default"
```

Comment:
`Target.RunParameterOrDefault` or `Target.RunOrDefault`?

Reply:
Nevermind

Reply:
Yes I like your suggestion...I will do it and normalize casing in a new PR, do you agree?

Reply:
actually I was wrong on this. I have no idea where this file is used and if updating this breaks anything...

Reply:
I was thinking you were saying that RunParameterOrDefault would be a better name than just RunOrDefault?

Reply:
No your change is fine. I'm not sure why we even have runparameterordefault